Source/Cells/BrickCell.swift

```diff
@@ -106,6 +106,7 @@ open class BaseBrickCell: UICollectionViewCell {
         // UICollectionView zIndex management 'fixes' the issue
         // http://stackoverflow.com/questions/12659301/uicollectionview-setlayoutanimated-not-preserving-zindex
         self.layer.zPosition = CGFloat(layoutAttributes.zIndex)
+        self.index = layoutAttributes.indexPath.item
```

Comment:
Unfortunately, it's not safe to assume that the `index` of a BrickCell is the same as the item of the indexPath.
If you have multiple bricks, but only 1 of them has a repeat count, it might not add up
For example:
```
BrickSection(bricks: [
  titleBrick,
  productBrick
])
```
So if product brick is repeated 5x, then the result would be:
```
titleBrick (indexPath item: 0 - index: 0)
productBrick (indexPath item: 1 - index: 0)
productBrick (indexPath item: 2 - index: 1)
productBrick (indexPath item: 3 - index: 2)
productBrick (indexPath item: 4 - index: 3)
productBrick (indexPath item: 5 - index: 4)
```
Please add a test that makes sure that this situation will work